scripts/build_deploy.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+
+declare -a enviroments=(
+# "<name>         <domain>                <legacy-domain>"
+  "ci-europa      ci-europa.kbase.us      legacy.ci-europa.kbase.us"
+  "narrative-dev  narrative-dev.kbase.us  legacy.narrative-dev.kbase.us"
```

Comment:
This appears to be missing
* legacy.ci.kbase.us
* legacy.next.kbase.us
* legacy.staging.kbase.us

Reply:
None of those environments exist at this time

Reply:
They'll need to, eventually but they're not there yet...